packages/protocol/src/utils/TimestampUtil.ts

```diff
@@ -10,6 +10,4 @@ export default function parse(millisOrString: number | string): number | never {
     }
 
     throw new Error(`Invalid timestamp: ${millisOrString}`)
-}
-
-export { parse }
```

Comment:
How the `parse` function should be visible for the users of `streamr-client-protocol` library? As a `TimestampUtil` function? (see export/import in `utils/index.ts`). Maybe removing this export doesn't do any regression, but it could make sense if this method would be available e.g. as `TimestampUtil.parse`.

Reply:
removed file since isn't used anywhere